src/components/LayoutContainer.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import {
+  Icon,
+  Image,
+  Menu,
+  Responsive,
+  Segment,
+  Sidebar,
+  Container
+} from "semantic-ui-react";
+
+import { Page } from "../constants";
+import logo from "../images/reel-politik-logo-1.png";
+import { string } from "prop-types";
+
+interface propTypes {
+  activePage: Page;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+const Logo = () => (
+  <Image src={logo} style={{ height: "3em", margin: "0.5em" }} />
+);
+
+const LayoutContainer: React.SFC<propTypes> = ({
```

Comment:
Overall I think I'm a bit confused about how this component is working with the mobile vs not mobile parts. I think trying to extract some of the duplication here and breaking it into smaller components would help

Reply:
AD, re-organized so that responsive is on top